lib/utils/money.ts

```diff
@@ -0,0 +1,25 @@
+import { FormID } from './form-data';
+
+interface IncomeInterface {
+  heading: string;
+  sections: Sections[];
+}
+
+interface Sections {
+  heading: string;
+  id: FormID;
+}
+
+export const Income: IncomeInterface = {
```

Comment:
Shouldn't need to redefine this, we can add to `getApplicationSectionsForResident`